cpp/tests/centrality/katz_centrality_test.cu

```diff
@@ -72,56 +77,55 @@ typedef struct Katz_Usecase_t {
 } Katz_Usecase;
 
 class Tests_Katz : public ::testing::TestWithParam<Katz_Usecase> {
- public:
+public:
   Tests_Katz() {}
   static void SetupTestCase() {}
   static void TearDownTestCase() {}
   virtual void SetUp() {}
   virtual void TearDown() {}
 
   void run_current_test(const Katz_Usecase& param) {
-       Graph_ptr G{new cugraph::Graph, Graph_deleter};
-       gdf_column_ptr col_src, col_dest, col_katz_centrality;
+    FILE* fpin = fopen(param.matrix_file.c_str(),"r");
```

Comment:
It would be great if we moved to the C++ style of file reading.

Reply:
I haven't focused on this while removing gdf_column, but I'd be happy with consistency between the tests.  We have many different ways that we use of loading data for the tests.

Reply:
> It would be great if we moved to the C++ style of file reading.

Would you expect a big performance difference or is it primarily motivated by having a more consistent C++ style?

Reply:
Note that we're using a set of functions in test_utils.h to manipulate the Matrix Market format.  This fopen call opens the file so we can pass the FILE * into the other test functions.  Changing to C++ style would require rewriting all of that code.

Reply:
> > It would be great if we moved to the C++ style of file reading.
> 
> Would you expect a big performance difference or is it primarily motivated by having a more consistent C++ style?

Just for consistency. Since we are trying to move (almost) everything in cugraph to RAII style it would be nice to take care of it in this cycle.

Reply:
@kaatish I'd suggest writing up an issue for this so it can get prioritized.  It would potentially affect all of the C++ unit tests.